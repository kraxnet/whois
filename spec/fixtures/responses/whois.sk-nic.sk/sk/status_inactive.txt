% For more information on Whois status codes, please visit https://icann.org/epp
%
% This WHOIS service is provided by SK-NIC and only contains
% information pertaining to Internet domain names registered by our
% our customers. By using this service you are agreeing (1) not to use any
% information presented here for any purpose other than determining
% ownership of domain names, (2) not to store or reproduce this data in
% any way without SK-NIC's approval, (3) not to use any high-volume, automated, electronic processes
% to obtain data from this service. Abuse of this service is monitored and
% actions in contravention of these terms will result in being permanently
% blacklisted.
%
% Access to the whois service is rate limited.

Domain:                       sekacnakolesach.sk
Registrant:                   ANNA-0426-3
Admin Contact:                ANNA-0426-3
Tech Contact:                 ANNA-0426-3
Registrar:                    SK-NIC
Created:                      2015-03-23
Updated:                      2017-09-01
Valid Until:                  2018-03-26
EPP Status:                   inactive

Registrar:                    SK-NIC
Organization:                 SK-NIC, a.s.
Organization ID:              35698446
Phone:                        +421.235035030
Email:                        hostmaster@sk-nic.sk
Street:                       Borská 6
City:                         Bratislava
Postal Code:                  84104
Country Code:                 SK
Created:                      2017-09-01
Updated:                      2017-09-11

Contact:                      ANNA-0426-3
Organization:                 Anna Škrabská
Name:                         Anna Škrabská
Organization ID:              43312489
Phone:                        +31.622994069
Email:                        ursulaahermina@hotmail.com
Street:                       Horné Pršany 125
City:                         Horné Pršany
Postal Code:                  97405
Country Code:                 SK
Registrar:                    SK-NIC
Created:                      2017-09-01
Updated:                      2017-09-01
