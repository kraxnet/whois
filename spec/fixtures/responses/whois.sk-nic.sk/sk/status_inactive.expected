#status
  %s == :registered

#available?
  %s == false

#response_throttled?
  %s == false

#registered?
  %s == true

#nameservers
  %s %CLASS{array}
  %s %SIZE{0}
